### DISCUSSION TITLE
Having issue with video writer

### DISCUSSION THREAD

**Discussion by SawantMyelin** [bluenviron/mediamtx#1183] [Questions]
Hi, I am trying to publish a video stream from opencv to the server, and i ran the dummy code that is provided with it and i am getting the error : Exception: can't open video writer . 
The code snippet is located here  https://github.com/aler9/rtsp-simple-server#from-opencv . Please help me out.

**Comment by aler9** [comment]
Hello, as written in the README, check that OpenCV has been compiled with GStreamer support:

```
python3 -c 'import cv2; print(cv2.getBuildInformation())'
```

Check that the output contains GStreamer: YES.

Otherwise you have to re-compile OpenCV by following instructions here:

https://github.com/aler9/rtsp-simple-server#from-opencv

[a comment by an automated account is not shown]
